### ISSUE TITLE
0crat failed Paypal payments

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#1867]
0crat failed to pay me. I've receive these messages in 0crat telegram chat last week:
- 16/01/2019 06:41 "We tried to pay your debt of $67.77, but failed (Failed to pay); don't worry, we will retry very soon"
- 17/01/2019 07:17 "We tried to pay your debt of $81.64, but failed (Failed to pay); don't worry, we will retry very soon"
- 18/01/2019: "We tried to pay your debt of $91.64, but failed (Failed to pay); don't worry, we will retry very soon"

Then 0crat said that it paid me:
19/01/2019 08:58 "We just paid you the debt of $91.64 (AP-5E652065BF505915F)"

But it did not paid me, just marked some eCheck in paypal, whose status is pending and the message is:
"eCheck estimated completion date is 24 January 2019. Do not ship items until you receive payment."

We're at 24/01/2019 and this echeck payment is still pending. 

Why zerocracy moved the payment to this new echeck payment type?

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by paulodamaso** [comment]
@g4s8 Received the payment some minnutes ago, thanks

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#1867` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope
